### DISCUSSION TITLE
After exporting static html, how can I judge whether to jump through Link or native a tag jump

### DISCUSSION THREAD

**Discussion by SpiritLing** [vercel/next.js#13999] [Help]
Condition: Export static html

How to judge whether to jump by next/link or direct access (that is, a tag access)

_Translated_ by Google

**Comment by rafaelalmeidatk** [comment]
Use the `next/link` component when the destination is a Next.js page (inside the `pages/` folder), and the `<a>` tag for everything else

**Comment by revskill10** [comment]
You just use `next/link` , Next.js will decide it's a native or SPA link based on your page is `static rendered` or `server side rendered`. You can try by yourself to see ?

**Comment by SpiritLing** [comment]
@revskill10 @rafaelalmeidatk 

Translated by Google

You may have misunderstood

What I need is this:
1. Enter `url` directly to enter the `index` page
2. Jump through `next/link` to enter the `index` page

Need to judge in the index page is the way to enter the above.

The corresponding purpose is to determine whether to load the `js` file to this page through `next/link` when using static `html`, or to enter this page through direct access.

`next` jumps in by loading the `js` file, so I think it should be able to tell which way it is.

If not, I hope this judgment can be added.

When I upload static html to `gh-pages` on `github`, this is a document content, and I need to handle different menu display by judging the way to enter the page.

**Reply by SpiritLing** [reply]
will use them, but I need to judge them

**Comment by SpiritLing** [comment]
Help Me !
